### ISSUE TITLE
What about the related parameters for NYUD and BIPED DATABASE?

### ISSUE THREAD

**Issue by k3chencoco** [PXinTao/XYW-Net#5]
I execute the codes of XYW-Net. It seems the loss can reach about 0.28 rapidly during traing with BSDS500. But the loss keeps about 0.5 after some epochs with BSDS500_PASCAL. besides, the loss keeps 1.0 after some epochs with NYUD. Can you share the parameters for the different DATABASEs?

**Comment by PXinTao** [comment]
In paper, we’ve described this in more detail. Here’s the relevant part:

We used the Adam optimizer with an initial learning rate of 1×10⁻³, a batch size of 1, and the input images at their original size. For the BSDS500 and BIPEDv1 datasets, the model was trained for 6 full iterations over the entire training data. Starting from the 6th iteration, the learning rate was reduced to 0.1 times the value from the previous iteration. For the NYUD and Multicue datasets, the model was trained for 20 iterations over all the training data, and the learning rate was reduced to 0.1 times the value from the previous iteration at the 10th and 15th iterations.

It’s important to note that the NYUD dataset is relatively complex and contains more noise, which could explain why the loss tends to stabilize at a higher value. This might be the reason you're seeing a higher loss during training. Hope this helps!

**Comment by PXinTao** [comment]
> I execute the codes of XYW-Net. It seems the loss can reach about 0.28 rapidly during traing with BSDS500. But the loss keeps about 0.5 after some epochs with BSDS500_PASCAL. besides, the loss keeps 1.0 after some epochs with NYUD. Can you share the parameters for the different DATABASEs?

In paper, we’ve described this in more detail. Here’s the relevant part:

We used the Adam optimizer with an initial learning rate of 1×10⁻³, a batch size of 1, and the input images at their original size. For the BSDS500 and BIPEDv1 datasets, the model was trained for 6 full iterations over the entire training data. Starting from the 6th iteration, the learning rate was reduced to 0.1 times the value from the previous iteration. For the NYUD and Multicue datasets, the model was trained for 20 iterations over all the training data, and the learning rate was reduced to 0.1 times the value from the previous iteration at the 10th and 15th iterations.

It’s important to note that the NYUD dataset is relatively complex and contains more noise, which could explain why the loss tends to stabilize at a higher value. This might be the reason you're seeing a higher loss during training. Hope this helps!

**Comment by k3chencoco** [comment]
got it. thanks very much. The parameters of these different DATABASES need to change are learing rate and epoch according to the paragrams.
********************************************************************************************************************************* 
We used the Adam optimizer with an initial learning rate of 1×10⁻³, a batch size of 1, and the input images at their original size. For the BSDS500 and BIPEDv1 datasets, the model was trained for 6 full iterations over the entire training data. Starting from the 6th iteration, the learning rate was reduced to 0.1 times the value from the previous iteration. For the NYUD and Multicue datasets, the model was trained for 20 iterations over all the training data, and the learning rate was reduced to 0.1 times the value from the previous iteration at the 10th and 15th iterations.
